chroma-common/chroma-common.spec

```diff
@@ -1,7 +1,7 @@
 %{!?name: %define name chroma-common}
-%{?!version: %define version %(%{__python2} -c "from chroma_common import version; sys.stdout.write(version())")}
+%{?!version: %define version %(%{__python} -c "from chroma_common import version; sys.stdout.write(version())")}
```

Comment:
I made the change to using __python2 in the previous patch but it's causing problems in this patch so I have reverted it to what is currently on master (no change). if we want to move to using __python2 in all the spec files we should do it in a separate patch.